examples/scripts/moo_parallel.py

```diff
@@ -15,7 +15,7 @@
 import torch
 
 from evotorch import Problem
-from evotorch.algorithms import SteadyStateGA
+from evotorch.algorithms import GeneticAlgorithm
```

Comment:
We need to go over all docs and examples (including the web page) and make sure we make appropriate changes for this renaming

Reply:
Thanks, agreed.